src/audio/aria/aria_hifi3.c

```diff
@@ -102,9 +105,13 @@ void aria_algo_get_data_odd_channel(struct processing_module *mod,
 			/*process data one by one if ch_n is odd*/
 			for (ch = 0; ch < ch_n; ch++) {
 				AE_L32_XP(in_sample, (ae_int32 *)in, inc);
-				out_sample = AE_MULFP32X2RS(in_sample, gain);
-
-				out_sample = AE_SLAA32S(out_sample, att);
+				out1 = AE_MUL32_HH(in_sample, gain);
+				out1 = AE_SRAA64(out1, shift_bits);
+				out1 = AE_SLAA64(out1, 16);
+				out2 = AE_MUL32_LL(in_sample, gain);
+				out2 = AE_SRAA64(out2, shift_bits);
+				out2 = AE_SLAA64(out2, 16);
+				out_sample = AE_ROUND32X2F48SSYM(out1, out2);
```

Comment:
sorry forgot to tell you if for the odd channel, only out1 is enough since the H&L part of in_sample is the same. it could be like:
```
				out1 = AE_MUL32_HH(in_sample, gain);
				out1 = AE_SRAA64(out1, shift_bits);
				out1 = AE_SLAA64(out1, 16);
                                out_sample = AE_ROUND32X2F48SSYM(out1, out1);
```